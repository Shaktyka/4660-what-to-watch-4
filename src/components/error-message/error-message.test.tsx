import * as React from 'react';
import * as renderer from 'react-test-renderer';

import ErrorMessage from './error-message';

describe(`ErrorMessage rendering`, () => {

  it(`ErrorMessage renders correctly`, () => {
    const message = `Текст ошибки`;

    const tree = renderer
      .create(
          <ErrorMessage
            message={message}
          />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

});
