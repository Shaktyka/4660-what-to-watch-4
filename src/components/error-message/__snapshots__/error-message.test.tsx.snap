// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage rendering ErrorMessage renders correctly 1`] = `
<div
  onClick={[Function]}
  style={
    Object {
      "backgroundColor": "brown",
      "borderRadius": "5px",
      "boxShadow": "0 0 10px 2px rgba(0, 0, 0, 0.2)",
      "boxSizing": "border-box",
      "color": "white",
      "cursor": "pointer",
      "fontSize": "16px",
      "maxWidth": "300px",
      "padding": "25px 30px",
      "position": "absolute",
      "right": "20px",
      "textAlign": "center",
      "top": "20px",
      "zIndex": 10,
    }
  }
>
  Текст ошибки
</div>
`;
